.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -38,48 +71,75 @@ def integration_steps():
         upload_coverage=True,
     ).get_tox_build_steps()
 
-    integration_suites_root = os.path.join(
-        SCRIPT_PATH, "..", "..", "..", "..", "integration_tests", "test_suites"
-    )
-    integration_suites = [
-        os.path.join("integration_tests", "test_suites", suite)
-        for suite in os.listdir(integration_suites_root)
-    ]
+    tests += build_spec_backcompat_suite()
+    tests += build_spec_celery_k8s_suite()
+    tests += build_spec_k8s_suite()
+    tests += build_spec_daemon_suite()
+
+    return tests
+
+
+def build_spec_backcompat_suite():
+    tox_env_suffix_map = {
+        "-old-dagit": {"dagit": "0.12.8", "user_code": "current_branch"},
+        "-old-user-code": {
+            "dagit": "current_branch",
+            "user_code": "0.12.8",
```

Comment:
It seems non-ideal to have to update this in code each week. Have you seen how we compute this based on packaging data in the internal repo? I think if we had a matrix like

{most recent major release (0.12.0), most recent minor release (0.12.8), master} that would give us pretty good coverage